WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -186,7 +235,7 @@ public class BlockEditorScreen: ScreenObject {
     private func addBlock(_ blockLabel: String) {
         addBlockButton.tap()
         let blockButton = app.buttons[blockLabel]
-        XCTAssertTrue(blockButton.waitForIsHittable(timeout: 3))
+        if !blockButton.isHittable { app.scrollDownToElement(element: blockButton) }
```

Comment:
some blocks are hidden from the initial view (e.g audio block) and require scrolling to get to it